GooglePlayInstant/Editor/AssetBundleVerifierWindow.cs

```diff
@@ -50,6 +50,13 @@ public static void ShowWindow()
         //TODO: Support Unity 5.6.0+
         private void StartAssetBundleVerificationDownload()
         {
+            if (string.IsNullOrEmpty(_assetBundleUrl))
```

Comment:
How can this happen? Should an exception or some other error be given so it fails before it gets to this point?

Reply:
This happens because the usual "Malformed URL" error is thrown in the line of code DownloadHandler.GetContent() in the get download info phase for some reason. However, because the _webrequest object is technically "done" the Update() function continously tries to complete the get download info phase, spitting out 999+ error messages to console with Malformed URL. 

I couldn't decide on where the appropriate place to catch this error was. I figured I wouldn't want it to start the download process /at all/ so it should be caught there.

As a quick note, it's completely valid  for the url to be null or a empty. This is a result of using the GUI and not filling in the assetBundle URL field. Would it be better if I caught this instead where the user clicks the verify button on the Quick Deploy window?

Reply:
Let's try to catch any obvious issues with malformed URLs before displaying the verify dialog and instead present a targeted error message dialog.

Reply:
Resolved.